docs/mcmc/index.rst

```diff
@@ -117,39 +120,41 @@ References
 Example
 =======
 
-.. note::
-
-   This example probably needs to be simplified to reduce the run time
-
 Simulate the data
 ------------------
 
-Create a simulated data set::
+.. plot::
+    :include-source:
+    :context:
+    :nofigs:
+
+    Create a simulated data set:
 
-    >>> np.random.seed(2)
+    >>> import numpy as np
+    >>> np.random.seed(2)  # For reproducibility of this example
```

Comment:
Ideally we'd do something like

```
rng = np.random.RandomState(2)  # For reproducibility of this example
```

and then use that below, but that's a larger change. We really should use `np.random.default_rng(seed)` but that's not guaranteed to be reproducible over time and we'd have to validate the results so using `RandomState` is easier for us...

Reply:
done.